dev/openchpl_data-model.sql

```diff
@@ -2866,6 +2866,56 @@ CREATE TABLE openchpl.filter (
         ON DELETE RESTRICT ON UPDATE CASCADE
 );
 
+CREATE TABLE openchpl.complaint_type (
+	complaint_type_id bigserial not null,
+	name text not null,
+	creation_date timestamp without time zone NOT NULL DEFAULT now(),
+    last_modified_date timestamp without time zone NOT NULL DEFAULT now(),
+    last_modified_user bigint NOT NULL,
+    deleted boolean NOT NULL DEFAULT false,
+	CONSTRAINT complaint_type_pk PRIMARY KEY (complaint_type_id)
+);
+
+CREATE TABLE openchpl.complaint_status_type (
+	complaint_status_type_id bigserial not null,
+	name text not null,
+	creation_date timestamp without time zone NOT NULL DEFAULT now(),
+    last_modified_date timestamp without time zone NOT NULL DEFAULT now(),
+    last_modified_user bigint NOT NULL,
+    deleted boolean NOT NULL DEFAULT false,
+	CONSTRAINT complaint_status_type_pk PRIMARY KEY (complaint_status_type_id)
+);
+
+CREATE TABLE openchpl.complaint (
+    complaint_id bigserial not null,
+    certification_body_id bigint not null,
+    complaint_type_id bigint not null,
```

Comment:
This can't be required, based on the line 

> And yes i would think that there needs to be a way of "clearing" that value if they accidentally selected a value